### ISSUE TITLE
Oké, en wat nu? 🧐

### ISSUE THREAD

**Issue by roelvangils** [WCAG-Audit-Discussions/NL-BE#1]
Ik vind het een supergoed idee van @rianrietveld om de conclusies van de vele discussies over WCAG-interpretaties en twijfelgevallen (eindelijk) ergens centraal te documenteren. Vaak kom je (al denkend en discussiërend) tot interessante nieuwe inzichten en soms verander je van gedacht door te luisteren naar de argumenten van anderen. Ook de discussies zelf zouden hier kunnen plaatsvinden, maar we moeten er misschien wel over waken dat we dan niet naast elkaar gaan praten (op de Fronteers #accessibility Slack wordt er ook lustig gediscussieerd).

Over het cerntraal bijhouden van conclusies over WCAG-vraagstukken hebben wij het in België ook al vaak gehad. We dachten aan een wiki of zo, maar eigenlijk is Github beter. 

Misschien moeten we wel een soort van simpele workflow uitwerken waarbij er (eventueel) ook een poll kan opgezet worden kan worden (in geval van onenigheid) of iets uiteindelijk een FAIL of een PASS wordt. Om dat goed te doen, moet elke "case" ook wel duidelijk gedocumenteerd worden natuurlijk.

Let wel: dit alles is louter informatief; auditeurs doen met deze informatie wat ze willen, maar in teams van testers is het wel héél belangrijk om consistentie in de beoordeingen te hebben (en te kunnen terugzoeken naar hoe daar in het verleden over gedacht werd). Het verhoogt ook de geloofwaardigheid.

**Comment by ShadowBB** [comment]
We zijn toevallig op dit moment bezig om dit op te zetten op internationaal vlak:
https://github.com/ShadowBB/WCAG_Audit_Examples

**Comment by julezrulez** [comment]
Ik zit al een tijdje te broeden op de structuur van vastleggen. Via welke insteek je de informatie terug wilt of kunt vinden.

- Op basis van **succescriterium** (zie ook [https://github.com/ShadowBB/WCAG_Audit_Examples])
- Op basis van **probleem** (bijvoorbeeld "Is het gebruik van maximum-scale in de meta-viewport-tag nog een reden om af te keuren?")

Het nadeel van vastleggen per succescriterium is:
- Je weet soms nog niet onder welk succescriterium het probleem past.
- Sommige vraagstukken vallen onder meerdere succescriteria.
- Soms willen we kunnen discussieren over een best practice die onder geen enkel succescriterium past.
- EN 301 549 heeft een eigen aanvulling op WCAG. Ook daar zullen vraagstukken uit voortvloeien.

Hoe verhoudt dit platform zich dan met WCAG Audit Examples?
- Streven is om hier snel tot een besluit te komen. Ik denk aan doorlooptijden van maximaal een week. (als dat haalbaar is??)
- Streven is om niet af te wijken van wat internationaal gangbaar is. Dus uitkomsten van vraagstukken zullen altijd gebaseerd zijn op de specs van onder andere W3C, 
- Discussies zullen hier (hoofdzakelijk?) in het Nederlands zijn. Dezelfde discussie kan en mag natuurlijk parallel elders plaatsvinden.
- Een goed voorbeeld wat hier ondergebracht kan worden zijn Nederlandse/Vlaamse/Friese/etc. voorbeeldteksten voor SC 3.3.1, 3.3.2 en 3.3.3. Paul van Buuren heeft al eens de behoefte geuit voor zo'n lijst met standaard Nederlandse voorbeeldteksten voor foutmeldingen en foutsuggesties.
- Ik zie dit discussieplatform als een goede basis om [WCAG Audit Examples](https://github.com/ShadowBB/WCAG_Audit_Examples) te voeden.
- Discussies kunnen op internationaal vlak doorgezet worden als we er nationaal niet uitkomen.

@ShadowBB 
- Zie jij redenen om niet een eigen "chapter" hier te beginnen?

**Comment by ShadowBB** [comment]
Hey Jules,

Ik heb ook nagedacht over deze indeling en ik kwam tot de conclusie dat altijd mijn eerste vraag is "Onder welk SC ben je aan het twijfelen dit af te keuren?" Dit maakt de discussie in theorie 50 keer zo makkelijk op niveau AA (hehe) maar in de praktijk vaak 2 of 3 keer zo simpel.

Als je eenmaal zeker weet dat je een bepaalde situatie zou moeten afkeuren onder een bepaald SC zegt dit ook nog eens niks over of je het niet ook op andere SC moet afkeuren. Dat zijn allemaal losstaande discussies.

Een vraagstuk valt dus ook nooit "onder meerdere SC".

Een best practice is nuttig om over te discussiëren, maar dat is niet de insteek van mijn specifieke Github Repository.

Dat er vraagstukken voortvloeien uit EN 301 549 vind ik juist eerder een argument voor het indelen op SC niveau, want dan kun je voor die specifieke aanvullingen ook een aparte pagina maken en dan vervuilt dat de pure WCAG discussie niet.

Je bullet points die duidelijk maken hoe dit platform zich zou verhouden tot WCAG Audit Examples zijn goed en duidelijk (hoewel ik bij WCAG Audit Examples ook niet af zou willen wijken van wat internationaal gangbaar is).

Dus ik zie geen enkele reden om niet een eigen chapter te beginnen en Nederlandse discussies te voeren die vervolgens WCAG Audit Examples voeden. Ik denk dat ik zelfs graag mee discussieer (zeker als dat betekent dat ik eigenlijk ook tegelijkertijd WCAG Audit Examples voed).

Ik zou wel aanraden om de discussies in te delen op SC niveau en dan simpelweg meerdere losstaande discussies te starten als je niet zeker weet onder welk SC iets afgekeurd moet worden. Maar ik zou er ook mee kunnen werken als dit platform anders ingedeeld zou worden.

**Comment by julezrulez** [comment]
Hi Brian, thanks.... je hebt me weer aan het denken gezet.

Ik neig nu ook meer naar de SC-indeling eventueel aangevuld met "EN 301 549", "onbekend" en "best practice".

Eenzelfde indeling betekent ook dat we een discussie vanuit hier makkelijker kunnen voortzetten op jouw Github repository. En de uitkomst daar eenvoudiger kunnen terugkoppelen hier.

Ik zou ook niet willen afwijken van wat internationaal gangbaar is. Ik kan me daarom voorstellen dat we hier voorlopig besluiten om iets zo te beoordelen totdat duidelijk is wat internationaal gangbaar is. Ik ga ervan uit dat de internationale discussie een iets langere doorlooptijd kent

**Comment by ShadowBB** [comment]
Daar ga ik ook vanuit ja. :) Top, ik kijk er naar uit!

**Comment by rianrietveld** [comment]
Wat dachten jullie van de volgende constructie:

- Elke nieuwe vraag of discussie wordt een issue.
- Je kunt aan een issue meerdere labels hangen. Labels kunnen zijn 
   - nummer(s) van de Success Criteria, er kunnen meerdere SC aan een issue worden gehangen
   - onderwerpen, bijv afbeeldingen, links, kopjes
- als we tot een conclusie gekomen zijn wordt de uitkomst als voorbeelden bij de WCAG_Audit_Examples gezet.
- als een antwoord onder meerdere SC valt, kan deze bij meerdere voorbeelden in WCAG_Audit_Examples worden opgenomen.
- In  WCAG_Audit_Examples staat bij elk voorbeeld een referentie naar de discussie (het issue).

Als dit jullie wat lijkt moeten we gaan kijken hoe we dit internationaal gaan regelen. Qua consensus. Wnat ik kan me voorstellen dat de internationale a11y community hier ook een plas over wil doen.

**Comment by rianrietveld** [comment]
cc @hvj

**Comment by rianrietveld** [comment]
cc @hidde

**Comment by rianrietveld** [comment]
Naast WCAG_Audit_Examples zouden we hier ook een specifiek NL deel kunnen maken, met voorbeelden van tekst op een 404 pagina, goede teksten voor foutmeldingen etc.

**Comment by ShadowBB** [comment]
==  er kunnen meerdere SC aan een issue worden gehangen ==

Dit zou ik dus willen ontwijken. Ik verwacht dat het beter werkt om slechts 1 discussie te voeren per issue. Elk SC is een losse discussie toegepast op elke situatie.

Afgezien van dit ene detail ben ik het eens met het voorstel. (en zelfs als de consensus is dat de community graag meerdere discussies wilt voeren onder 1 issues dan kom ik daar wel uit.)

**Comment by rianrietveld** [comment]
@ShadowBB , ik heb zo een call met @julezrulez. Heb je zin om aan te haken?

**Comment by ShadowBB** [comment]
@rianrietveld Sure. Hoe?

**Comment by rianrietveld** [comment]
@ShadowBB kun je je e-mailadres/telefoon even mailen naar rian@level-level.com?

**Comment by julezrulez** [comment]
We zijn begonnen. Volgens mij kan hiermee dit issue gesloten worden. Ik zie in de discussie overigens wat informatieve teksten staan die we ook voor de website kunnen gebruiken.

**Comment by ShadowBB** [comment]
Eens dat dit gesloten kan worden. :)

**Comment by rianrietveld** [comment]
@ShadowBB ik wil de procedure aanpassen van het sluiten van issues. Nu is er geen zicht op wat er aan de website toegevoegd kan worden.

Graag het issue niet sluiten, maar er het label: "Toevoegen aan website" aanhangen. Dan kan degene die de conclusie aan de website heeft toegevoegd het issue sluiten is is het echt afgerond.

Ik open de issues weer en hang er het label aan, dan kan ik als ik tijd heb de website bijwerken.

 cc @julezrulez @hidde

**Comment by ShadowBB** [comment]
That makes a lot of sense!
